tests/e2e/vmgroups_test.go

```diff
@@ -111,6 +111,9 @@ func (vg *VmGroupTest) TearDownSuite(c *C) {
 	out, err = ssh.InvokeCommand(vg.config.EsxHost, cmd)
 	log.Printf(out)
 
+	// Remove Config DB
+	adminutils.ConfigRemove(vg.config.EsxHost)
+
```

Comment:
I don't follow this. What's the default customer scenario we are expecting? I may be wrong , but I think we are expecting a typical customer environment will be a clean vDVS installation without initializing local or shared DB. So most of e2e tests should be run in the default setup, i.e. NotConfigured mode.

Multi-tenancy features are experimental. So only VmGroup related tests should take care of ConfigInit and ConfigRemove.